externals/kyuubi-spark-sql-engine/pom.xml

```diff
@@ -167,8 +167,18 @@
                         <includes>
                             <include>org.apache.kyuubi:kyuubi-common</include>
                             <include>org.apache.kyuubi:kyuubi-ha</include>
+                            <include>org.apache.curator:*</include>
                         </includes>
                     </artifactSet>
+                    <relocations>
+                        <relocation>
+                            <pattern>org.apache.curator</pattern>
+                            <shadedPattern>${kyuubi.shade.packageName}.curator</shadedPattern>
+                            <includes>
+                                <include>org.apache.curator.**</include>
```

Comment:
what about the transitive dependencies, shall we explicitly exclude them or add them here too? cc @pan3793